### ISSUE TITLE
aws_volume_attachment.disk_attachment: doesn't support update

### ISSUE THREAD

**Issue by evgeniagusakova** [terraform-providers/terraform-provider-aws#2520]
Hi!

### Terraform Version
```
terraform  version
Terraform v0.11.0
+ provider.aws v1.5.0
+ provider.null v1.0.0
+ provider.template v1.0.0
```
### OS Version:
```
 uname -m -p -r -s -v
Darwin 17.2.0 Darwin Kernel Version 17.2.0: Fri Sep 29 18:27:05 PDT 2017; root:xnu-4570.20.62~3/RELEASE_X86_64 x86_64 i386
```

### Issue explanation:
Because of issue: https://github.com/terraform-providers/terraform-provider-aws/issues/1991 
I added to `skip_destroy = true` to my aws_volume_attachment deffinition.

Run plan: (some output is skipped but updated only one resource)
```
  ~ module.test_only_node.aws_volume_attachment.disk_attachment
      skip_destroy:             "" => "true"

```
But when I tried to apply changes I got an error:
```
Error: Error applying plan:
1 error(s) occurred:
* module.test_only_node.aws_volume_attachment.disk_attachment: 1 error(s) occurred:
* aws_volume_attachment.disk_attachment: doesn't support update
```

Expected behavior: 
`terraform plan` should notify about update is not possible
OR
`terraform apply ` should be able to update resource
OR
documentation here 
https://www.terraform.io/docs/providers/aws/r/volume_attachment.html 
should be updated and notifies that update is not supported for aws_volume_attachment resources. 

BTW it looks like option  ` skip_destroy` make sense only  on destroy phase and it is just flag in Terraform state so I do not understand reason why it can't be updated.

As far as I can understand what I tried to do is only modify Terraform state but not actually update AWS resources.

**Comment by apparentlymart** [comment]
Hi @evgeniagusakova! Sorry for this strange behavior.

Indeed it seems like something strange happened here. This resource doesn't support updating, so all of its attributes should currently be marked as requiring a new resource; it's strange that the resource's tests are passing since this is supposed to be caught by the "internal validate" step. :thinking: 

But with that said, you're also right that `skip_destroy` does not require any changes remotely because it's a Terraform-use-only attribute, so I think the right fix here is to actually _implement_ an `Update` function for this resource that does _nothing_, thus allowing Terraform's default behavior to update both `skip_destroy` and `force_detach`, while keeping the other attributes all marked as forcing a new resource so that they will continue to be handled via `Delete`+`Create`.

**Comment by bflad** [comment]
Hi @evgeniagusakova, sorry you're having trouble with this and thank you for reporting it. There is already an open issue for this bug (#1211) so in order to consolidate any discussions or fix efforts, I'm going to close this issue in preference of the earlier issue. I'd recommend adding 👍  to the other original issue comment as that's one way we help prioritize fixes across the provider.

**Comment by bflad** [comment]
This has been released in [terraform-provider-aws version 1.9.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#190-february-09-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
